README.md

```diff
@@ -22,7 +22,14 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+```
+elasticsearch:
+  url: "http://localhost:9200/" # Required. Supports auth and SSL: https://user:pass@someurl.com
+  number_of_shards: 1           # Optional. Default is 1 primary shard.
+  number_of_replicas: 1         # Optional. Default is 0 replicas.
```

Comment:
We start with 0 replicas for the initial bulk update, but should update to this value when we're done.